SConstruct

```diff
@@ -5,5 +5,5 @@ scripts.BasicSConstruct(
     "sdm_schemas",
     disableCc=True,
     noCfgFile=True,
-    versionModuleName="python/lsst/sdm_schemas/version.py"
+    versionModuleName="python/lsst/sdm/schemas/version.py"
```

Comment:
I think this is the default so things should work if you remove the line.

Reply:
I was planning to update this so that the `version.py` file is not needed.

Reply:
Changed to use versioning without `version.py`.